src/app/auth/auth/auth.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="auth-conainer">
+  <div class="auth-conainer__main mat-elevation-z12">
+    <mat-card>
+      <mat-card-title>User login</mat-card-title>
+      <mat-card-content class="auth-conainer__card-content">
```

Comment:
please also set background of the window (not dialog) to gray color -- same as we have in app